hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,68 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <br/>
+     * If set, subsequent consensus_update_topic transactions signed with this
+     *  key MAY update or delete the custom fees for this topic.
+     * <p>
+     * If set to a Key containing a non-empty KeyList, the previous key and new
+     *  key MUST both sign this transaction. The previous value will be replaced.
+     * <br/>
+     * If set to a Key containing an empty KeyList, the previous key and the
+     *  admin key MUST both sign this transaction. The previous value will be
+     *  cleared.
+     * <br/>
+     * If not set, the current key SHALL NOT change.<br/>
+     * <br/>
+     * If unset in state, this field MUST NOT be set in this transaction.
+     * <br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * <br/>
```

Comment:
There seems to be a confusion about when to add a paragraph.
The paragraph separates _description_ from _requirement_.
This doesn't do that (also, it adds newlines in a lot of unnecessary places, that make this hard to read inline).
```suggestion
     * Access control for update/delete of custom fees.<br/>
     * <p>
     * If set, subsequent consensus_update_topic transactions signed with this
     * key MAY update or delete the custom fees for this topic.<br/>
     * If set to a Key containing a non-empty KeyList, the previous key and new
     * key MUST both sign this transaction.
     * If this field is set to a valid `Key`, the previous value
     * SHALL be replaced.<br/>
     * If set to a `Key` containing only an empty `KeyList`, the previous key
     * and the admin key MUST both sign this transaction.<br/>
     * If set to a `Key containing only an empty `KeyList`, the previous
     * value SHALL be cleared.<br/>
     * If not set, the current key SHALL NOT change.<br/>
     * If unset in state, this field MUST NOT be set in this transaction.<br/>
     * If not set when the topic is created, this field CANNOT be
     * set via update.<br/>
```

Reply:
Fixed